src/packages/solid/components/Checkbox/Checkbox.types.ts

```diff
@@ -0,0 +1,55 @@
+import type { NodeProps, NodeStyles } from '@lightningjs/solid';
+import type { ComponentStyleConfig, NodeStyleSet, Tone } from 'types/types.js';
+import type { UIComponentProps } from 'types/interfaces.js';
+
+export interface CheckboxProps extends UIComponentProps, CheckboxStyleProperties {
+  /**
+   * Indicates whether the checkbox is checked or unchecked.
+   * Setting this to `true` will check the checkbox, and setting it to `false` will uncheck it.
+   */
+  checked?: boolean;
+  children?: NodeProps['children'];
+}
+
+export interface CheckboxStyleProperties {
```

Comment:
can we add core reference links for the rest of the props here? i know it's annoying, but one of the issues with aliasing properties like this is that we lose the reference to the original prop. its a stop-gap, but it does help